version.config

```diff
@@ -1,4 +1,3 @@
 version = 2.3.1
 update_from_version = 2.3.0
 downgrade_to_version = 2.3.0
-
```

Comment:
What is this commit for?

Reply:
Seems to be remains from an update patch from master. Removed it.